### ISSUE TITLE
with CRON like pattern, job just stopped with no error after 25d 

### ISSUE THREAD

**Issue by lvgithub** [kelektiv/node-cron#388]
```
const CronJob = require('cron').CronJob;
const cronJob1 = new CronJob('*/5 * * * * *', async function () {
    const ret = await redis.lock('LOCK_REFRESH_WIDGET', 4);
    if (ret) {
        await refresh();
    }
});
cronJob1.start();
```

_Originally posted by @lvgithub in https://github.com/kelektiv/node-cron/issues/387#issuecomment-439752249_

**Comment by cuongtb-ibl** [comment]
yes, same here
my setting of cronjob is * * * * * * and it's stopped after 2 days without error

**Comment by ncb000gt** [comment]
@cuongtb-ibl do you have a code sample you can supply?

**Comment by cuongtb-ibl** [comment]
`const timer = new cron.CronJob({
            cronTime: f.time, // * * * * * *
            onTick: async () => {
                try {
                    logger.debug(`start ${f.constructor.name} , time ${f.time}`);
                    timer.stop();
                    await f.start()
                    timer.start();
                    logger.debug(`${f.constructor.name} done!! , at ${new Date()}`);
                } catch (error) {
                    logger.error('have error job ', f.constructor.name + ' stop !!!', error);
                    timer.stop();
                }
            },
            start: false,
        });
        timer.start();`

@ncb000gt thanks

**Comment by ncb000gt** [comment]
Hmm. I'm wondering if it's the async aspect. Can you try removing that and see if it works?

**Comment by cuongtb-ibl** [comment]
thanks, I will try and come back to you

**Comment by pqvst** [comment]
It seems this is a bug that was fixed in Node 10.9.0
https://github.com/nodejs/node/issues/22149

**Comment by ncb000gt** [comment]
@pqvst awesome, thanks! i'm going to close this ticket. please let me know if you're still running into this.
